source/common/http/http1/codec_impl.cc

```diff
@@ -387,13 +387,14 @@ void ConnectionImpl::completeLastHeader() {
   ENVOY_CONN_LOG(trace, "completed header: key={} value={}", connection_,
                  current_header_field_.getStringView(), current_header_value_.getStringView());
 
+  auto& current_headers = headersOrTrailers();
```

Comment:
this is current headers or trailers as well, no?   Can you scan through and update naming while you're in here?

Reply:
Yes will do.